airflow/dags/ingestion/ask-astro-forum-load.py

```diff
@@ -15,7 +15,7 @@
 
 default_args = {"retries": 3, "retry_delay": 30}
 
-schedule_interval = "0 5 * * *" if ask_astro_env == "prod" else None
+schedule_interval = os.environ.get("INGESTION_SCHEDULE", "0 5 * * 2") if ask_astro_env == "prod" else None
```

Comment:
Would there be a possibility that we want to run these DAGs at different schedules? Might make sense to have separate schedule env vars for each of the DAG just in case :)